frontend/src/muralForm/MuralForm.tsx

```diff
@@ -87,22 +102,30 @@ function MuralForm({ handleCancel }: IMuralFormProps) {
       alert("Please check validity of address.");
       return;
     }
-    axios
-      .post(CREATE_MURAL_API, {
-        name: name,
-        boroughId: borough,
-        artistId: artist,
-        year: year,
-        city: "Montreal",
-        longitude: addressCoords[0],
-        latitude: addressCoords[1],
-        assistants: assistants,
-        partners: partners,
-        description: description,
-        socialMedia: socialMedia,
-        address: address,
-        neighbourhood: neighbourhood,
-      })
+
+    let payload = {
+      name: name,
+      boroughId: borough,
+      artistId: artist,
+      year: year,
+      city: "Montreal",
+      longitude: addressCoords[0],
+      latitude: addressCoords[1],
+      assistants: assistants,
+      partners: partners,
+      description: description,
+      socialMedia: socialMedia,
+      address: address,
+      neighbourhood: neighbourhood,
+    } as any;
+    let existingMural = mural && Object.keys(mural);
+    if (existingMural) payload.id = mural.id;
+    axios({
+      method: existingMural ? 'put' : 'post',
+      url: existingMural ?
+        `${CREATE_MURAL_API}/${payload.id}` : CREATE_MURAL_API,
+      data: payload
```

Comment:
If I understand, PUTing will pass every attribute of the mural, including ones that aren't updated? I think the endpoint will also work if you pass only the changed attributes, reducing writes to the database. Not a priority.